packages/marshal/docs/smallcaps-cheatsheet.md

```diff
@@ -4,19 +4,19 @@ An example-based summary of the Smallcaps encoding
 
 | Passable value  | OCapn name    | JS example          | JSON encoding        |
 | ----------------|---------------|---------------------|----------------------|
-| bigint          | SignedInteger | `7n`<br>`-7n`       | `"+7"`<br>`"-7"`     |
+| bigint          | Integer       | `7n`<br>`-7n`       | `"+7"`<br>`"-7"`     |
 | manifest constant | Undefined<br><br>Float64<br><br><br> | `undefined`<br>`Infinity`<br>`-Infinity`<br>`NaN`<br>`-0` | `"#undefined"`<br>`"#Infinity"`<br>`"#-Infinity"`<br>`"#NaN"`<br>`"#-0"` // unimplemented |
 | passable symbol | Symbol        | `Symbol.for('foo')`<br>`Symbol.asyncIterator` | `"%foo"`<br>`"%@@asyncIterator"` |
-| remotable       | Capability    | `Far('foo', {})`    | `"$0.foo"`           |
-| promise         | Capability    | `Promise.resolve()` | `"&1"`               |
+| remotable       | Remotable     | `Far('foo', {})`    | `"$0.foo"`           |
+| promise         | Promise       | `Promise.resolve()` | `"&1"`               |
 | special string  | String        | `'#foo'`            | `"!#foo"`            |
 | other string    | String        | `'foo'`             | `"foo"`              |
 | other JSON scalar | Null<br>Boolean<br><br>Float64 | `null`<br>`true`<br>`false`<br>`7.1` | `null`<br>`true`<br>`false`<br>`7.1` |
-| copyArray       | Sequence      | `[a,b]`             | `[<a>,<b>]`          |
+| copyArray       | List          | `[a,b]`             | `[<a>,<b>]`          |
 | copyRecord      | Struct        | `{x:a,y:b}`         | `{<x>:<a>,<y>:<b>}`  |
 | error           | Error         | `TypeError(msg)`    | `{"#error":<msg>,"name":"TypeError"}` |
 | tagged          | Tagged        | `makeTagged(t,p)`   | `{"#tag":<t>,"payload":<p>}` |
-| ? | ByteString | ? | ? |
+| ?               | ByteArray     | ?                   | ? |
```

Comment:
doesn't match https://github.com/ocapn/ocapn/wiki/Abstract-Syntax#bytestrings

I don't know what document is supposed to represent "latest agreed ocapn terminology".

Reply:
https://cryptpad.fr/code/#/2/code/edit/Ng7kXl9rCF5qnTQX60UCpDHE/ ?

Reply:
In which case, @kriskowal , that wiki doc should be updated too. It is a wiki, so I'll do it, depending on this thread to have brought it to your attention so you can correct my update if wrong.

Reply:
Fixed the wiki doc with a link into the mtg notes. Added to this doc a link to the wiki doc. Resolving.